libs/walproposer/src/walproposer.rs

```diff
@@ -197,18 +197,19 @@ impl Wrapper {
             .unwrap()
             .into_raw();
 
-        let mut safekeepers_list_vec = CString::new(config.safekeepers_list.join(","))
+        let mut safekeeper_connstrings_vec = CString::new(config.safekeeper_connstrings.join(","))
```

Comment:
The PR desc sayth: “A future PR will remove the neon.safekeepers GUC.” Here it looks like this PR is already removing compat with the previous name of the GUC?